README.md

```diff
@@ -4,34 +4,31 @@ An [awesome](https://github.com/sindresorhus/awesome)-style list of cool Bevy pr
 
 ## Learning
 
+### Tutorials
```

Comment:
Before making the final migration, we should remove the entries from the readme, explain the general concept, describe the toml format, and link to Bevy Assets.

Reply:
yup, kept it for now in case I want to rerun migration to add a field to every entry or something

Reply:
added in tracking issue